### PR TITLE
[REF] Skip builds if commit message has [ci skip]

### PR DESCRIPTION
**Pull request by hugho-ad**
FIX https://github.com/OCA/runbot-addons/issues/116

### CODE REVIEW

**Comment by hugho-ad** [comment]
@moylop260 what do you think ?

File: runbot_travis2docker/models/runbot_build.py [hunk 1]
```diff
@@ -194,3 +186,11 @@ def _local_cleanup(self, cr, uid, ids, context=None):
             if build.docker_container:
                 run(['docker', 'rm', '-f', build.docker_container])
                 run(['docker', 'rmi', '-f', build.docker_image])
+
+    def skip_check(self, build):
```
Comment (moylop260) [inline runbot_travis2docker/models/runbot_build.py]: You are receiving a parameter build but I didn't see a value sent from invoke

File: runbot_travis2docker/models/runbot_build.py [hunk 2]
```diff
@@ -125,11 +121,7 @@ def job_30_run(self, cr, uid, build, lock_path, log_path):
         if not build.branch_id.repo_id.is_travis2docker_build:
             return super(RunbotBuild, self).job_30_run(
                 cr, uid, build, lock_path, log_path)
-        if not build.docker_image or not build.dockerfile_path \
-                or build.result == 'skipped':
-            _logger.info('docker build skipping job_30_run')
-            return MAGIC_PID_RUN_NEXT_JOB
-
+        build.skip_check()
```
Comment (moylop260) [inline runbot_travis2docker/models/runbot_build.py]: You are returning a value but is not processed

**Review by moylop260 — CHANGES_REQUESTED** [review]
Could you add a test creating a build with the subject "ci-skip" please?

File: runbot_travis2docker/models/runbot_build.py [hunk 3]
```diff
@@ -194,3 +192,11 @@ def _local_cleanup(self, cr, uid, ids, context=None):
             if build.docker_container:
                 run(['docker', 'rm', '-f', build.docker_container])
                 run(['docker', 'rmi', '-f', build.docker_image])
+
+    def skip_check(self, cr, uid, build, context):
```
Comment (moylop260) [inline runbot_travis2docker/models/runbot_build.py]: assign None by default to context parameter

File: runbot_travis2docker/tests/test_runbot_build.py [hunk 4]
```diff
@@ -116,6 +116,40 @@ def test_jobs(self):
         self.assertEqual(
             self.build.result, u'ok', "Job result should be ok")
 
+    def test_jobs_ci_skip(self):
+        """Test the [ci skip] feature"""
+        'Create build and run all jobs'
```
Comment (moylop260) [inline runbot_travis2docker/tests/test_runbot_build.py]: Use a comment or a docstring
This is a unassigned string

**Comment by moylop260** [comment]
Could you force another commit in order to force build in travis?

**Comment by hugho-ad** [comment]
@moylop260 travis green

**Review by moylop260 — APPROVED** [review]
Thanks

**Comment by moylop260** [comment]
@lasley @pedrobaeza @alan196 your feedback is welcome

**Review by lasley — APPROVED** [review]
Neat, this will be great for docs branches 👍

**Review by pedrobaeza — APPROVED** [review]
And for Transifex commits (when I implement it that change)

**Comment by hugho-ad** [comment]
@moylop260 this pr can be merged ?

**Pull request merged**